proposals/array_modify.json

```diff
@@ -1,15 +1,15 @@
 {
     "id": "array_modify",
     "summary": "Change the content of an array (insert, remove, update)",
-    "description": "Allows to insert into, remove from or update an array.\n\nAll labels get discarded and the array indices are always a sequence of numbers with the step size of 1 and starting at 0.",
+    "description": "Allows to insert into, remove from or update an array. This process describes updating an array as two separate operations that first removes and then inserts elements.\n\nAll labels get discarded and the array indices are always a sequence of numbers with the step size of 1 and starting at 0.",
```

Comment:
> "summary": "Change the content of an array (insert, remove, update)",
>  "description": "Allows to insert into, remove from or update an array. This process describes updating an array as two separate operations that first removes and then inserts elements.

I think there is some room for finetuning here (related to defining update as remove+insert), e g.

> "summary": "Change the content of an array (remove, insert, update)",
> "description": "Modify an array by removing, inserting or updating items. Updating can be seen as removing items followed by inserting new items (not necessarily the same number).

Reply:
Updated, with the minor change that I used "elements" instead of "items" to be consistent.